Ports/iOSPort/nativeSources/CodenameOne_GLAppDelegate.m

```diff
@@ -589,11 +594,6 @@ -(void)application:(UIApplication*)application didChangeStatusBarFrame:(CGRect)o
 
 - (void)application:(UIApplication*)application didReceiveLocalNotification:(UILocalNotification*)notification {
     CN1Log(@"Received local notification while running: %@", notification);
```

Comment:
`application:didReceiveLocalNotification:` is still required for iOS <10.    `userNotificationCenter:didReceiveNotificationResponse:withCompletionHandler:` is only available in iOS >=10.

Hence, this change will break local notifications for iOS<10.

What device are you testing on?

Reply:
You're absolutely right, I had a feeling it would be the case but couldn't test because I only have iOS 10+ devices (iPad 4 & iPhone 6s)